src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx

```diff
@@ -62,7 +62,11 @@ export const DefaultHeaderArtworkFragmentContainer = createFragmentContainer(
         href
         slug
         image {
-          resized(width: 300, height: 450) {
+          resized(
+            width: 300
+            height: 450
+            version: ["normalized", "larger", "large"]
```

Comment:
TIL this can be an array??

Reply:
[Unfortunately.](https://github.com/artsy/metaphysics/blob/5f55ee918b29767d16fc915582195c5627455d6c/src/schema/v2/image/normalize.ts#L19-L35) — the image versions are relatively unreliable — the response might be missing a few.